config/locales/en.yml

```diff
@@ -2302,146 +2302,107 @@ en:
-
-        <p>If you have any questions, comments, concerns, or complaints with the Site, please contact us by email at %{email_link} or by mail at:</p>
-
-        <p>
-          GetCTC<br/>
-          c/o Code for America<br/>
-          972 Mission Street<br/>
-          San Francisco, CA 94103
-        </p>
-
-        <p>We will do our best to resolve the issue.</p>
+      privacy_policy:
+        01_intro:
+        - GetCTC.org is a service created by Code for America Labs, Inc. ("Code for America", "we", "us", "our") to help low to moderate-income households access tax benefits, like the Advance Child Tax Credit (AdvCTC) and Economic Impact Payments through accessible tax filing services.
+        - This Privacy Policy describes how we collect, use, share, and protect your personal information. By using our Services, you agree to the terms in this Privacy Policy. This Privacy Notice applies regardless of the type of device you use to access our Services.
+        02_questions_html: If you have any questions or concerns about this Privacy Notice, please contact us at %{email_link}
+        03_overview: Overview
+        04_info_we_collect: Information we collect
+        05_info_we_collect_details: 'We follow the Data Minimization principle in the collection and use of your personal information. We may collect the following information about you, your dependents, or members of your household:'
+        06_info_we_collect_list:
+        - Personal identifiers such as name, addresses, phone numbers, and email addresses
+        - Date of birth
+        - Demographic information, such as age and marital status
+        - Personal identifiers such as name, addresses, phone numbers, and email addresses
+        - Date of birth
+        - Demographic information, such as age and marital status
+        - Characteristics of protected classifications, such as gender, race, and ethnicity
+        - Tax information, such as social security number or individual taxpayer identification number (ITIN)
+        - State-issued identification, such as driver’s license number
+        - Financial information, such as employment, income and income sources
+        - Banking details for direct deposit of refunds
+        - Details of dependents
+        - Household information and information about your spouse, if applicable
+        - Information from your computer or device, such as IP address, operating system, browser, date and time of visit, and click-stream data (the website or domain you came from, the pages you visit, and the items you click on during your session)
+        07_info_required_by_irs: We collect information as required by the IRS in Revenue Procedure “Rev RP-21-24.”
+        '08_how_we_collect': How we collect your information
```

Comment:
Not sure why they want us to add quotes to this key name, but the normalize check fails unless these are here @alaroia-cfa

Reply:
@embarnard Curious, and OK!